package.json

```diff
@@ -17,11 +17,13 @@
   "author": "Andela Simulations Programme",
   "license": "MIT",
   "dependencies": {
+    "bcrypt": "^3.0.6",
     "body-parser": "^1.18.3",
     "cors": "^2.8.4",
     "dotenv": "^6.2.0",
     "errorhandler": "^1.5.0",
```

Comment:
This should not be here. We're not using it.

Reply:
@eokwukwe i want to be sure, are you referring to the `errorhandler` package??

Reply:
yes

Reply:
ok, done

Reply:
Edit the `.hound.yml` file. Add a colon after `eslint`. It should look like this:
```
eslint:
 enabled: true
 config_file: .eslintrc
 ignore_file: .eslintignore
```

Reply:
ok

Reply:
done